modules/apps/powwow/powwow-portlet/docroot/meetings/js/recurrence.js

```diff
@@ -699,4 +699,4 @@
 			requires: ['aui-base', 'aui-datatype', 'liferay-meeting-calendar-recurrence-util']
 		}
 	);
-}());	
```

Comment:
It's good to format source code to keep consistency in convention. But if changes are really just source formatting, there is one practice should come in mind:
- If you do not change any logic or business, do not have files included in a specific commit.
- If you want to include this kind of change in this PR, have a separate commit with desc `Source formatting` or simply `SF`

Anyhow, this change is very small so that it's fine to include in this commit.